### PR TITLE
Display cozy's notification as android notifications

### PR DESCRIPTION
**Pull request by jacquarg**
Use background service to synchronise notifications periodically (15'), use in progress js-background-service ( https://github.com/jacquarg/cordova-jsbackgroundservice )

There is stil some QD choices which should be discussed :
- For testing purpose, _forward_to_mobile_ filter is disconnected.
- Notifications are _removed_ from local db to be marked as read. There isn't synchronization with the cozy yet.
- Architecture isn't that clean (db requests directly in views/notifications.coffe, ...)
- Needs day-to-day testing, to test battery consumption, and robustness.

Labels : IN PROGRESS

### CODE REVIEW

**Comment by frankrousseau** [comment]
@poupotte @aenario who is going to review that PR?

**Comment by frankrousseau** [comment]
@jacquarg Can you add comments explaining what does each function?

File: www-src/app/service.coffee [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            @polyglot.extend locales
+            window.t = @polyglot.t.bind @polyglot
+
+
+            @replicator = new Replicator()
+            @replicator.init (err, config) =>
+                console.log "Service - Replicator inited"
+                if err
+                    console.log err, err.stack
+                    window.service.workDone()
```
Comment (aenario) [inline www-src/app/service.coffee]: should be a return or an else if below

**Comment by aenario** [comment]
Not much to say code-wise.

Architecture wise, I feel it would be cleaner to have the Service extends Backbone.Model. This class would then manage the JSBackgroundService and have daemonRunning as an attributes.
ConfigView would then only 
```service.on "change", -> render() ```

This would avoid the need for checkRunning every time you try to change it. And provide a single place for changes related to your plugin.

Do you want me to review the JSBackgroundPlugin or do you plan on tidying it up first ?

**Comment by aenario** [comment]
Also, from a non-technical user point of view, "background daemon" is scary :)

**Comment by frankrousseau** [comment]
Don't be afraid to merge it. We'll do a hotfix or a revert if it doesn't work.

**Comment by poupotte** [comment]
Actually, this issue requires test on long time. Moreover, notifications for isen aren't available on cozy platform for the moment. So, we have decided to merge the two others PR this week (to publish a new version on Friday). Then, we can test this PR (this week-end and next week) to publish it on the end of next week.

**Comment by frankrousseau** [comment]
Ok

**Comment by jacquarg** [comment]
@aenario, I think the plugins are ready to review now. Could you take a look at them (and tell me if they need more documentation) ? 
Biggest part here : https://github.com/jacquarg/cordova-jsbackgroundservice 
And the dessert : https://github.com/jacquarg/cordova-jsbgservice-newpicture

**Comment by jacquarg** [comment]
@aenario : last two commit resolve two bugs not related to service.
First : fix incompatibility of cozy-lobile with lower case doctype fields, and old request.
Second : fix photo folder spoofing, even if errors occurs on first initialization (typically on weak GSM).

File: www-src/app/locales/en.coffee [hunk 2]
```diff
@@ -98,3 +99,4 @@ module.exports =
     "An error happened (INVALID URL)": "An error occured (invalid url)."
     "This file isnt available offline": "This file isn't available offline."
     "ABORTED"               :  "The procedure was aborted."
+    "photo folder not replicated yet" : "Initialisation not finished yet."
```
Comment (poupotte) [inline www-src/app/locales/en.coffee]: InitialiZation

**Comment by poupotte** [comment]
It looks ok for me.

**Pull request merged**